cardano_node_tests/tests/test_plutus_mint_build.py

```diff
@@ -47,6 +47,23 @@ def payment_addrs(
     return addrs
 
 
+def _check_plutus_cost(
+    plutus_cost: list, expected_tx_time: int, expected_tx_space: int, expected_lovelace: int
+):
+    """
```

Comment:
We use google style doctstrings - https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
```python
"""Check plutus transaction cost.
...
"""
```
Also from the style guide - https://google.github.io/styleguide/pyguide.html#383-functions-and-methods
> The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable.""").